ddpui/datainsights/insights/insight_interface.py

```
@@ -131,5 +131,5 @@ def __init__(
         self.db_table: str = db_table
         self.db_schema: str = db_schema
         self.filter = filter_
-        self.wtype = wtype if wtype else WarehouseType.POSTGRES  # default postgres
+        self.wtype = wtype if wtype else e.POSTGRES  # default postgres
```

Ruff (0.14.0)

Undefined name `e`

(F821)